README.md

```diff
@@ -1,4 +1,10 @@
 # nodejs-collection
 A place for the Node.js Collective to organize the editing process
 
+## Tone and Content
+
+We are looking for posts that inform, teach, experiment, and generally enrich our community. There's no hard and fast rule of what an article should or feel like; Node.js Collection articles should be as diverse as the community itself. Ideally, contributions should encourage readers to explore new ideas, dig deeper into old ones, and to inspire readers to experiment and tinker. Articles can cover any number of topics such as:
+
+If you're passionate about something not listed above, please don't hesitate to submit your idea for consideration.
```

Comment:
Nit: `passionate` is a bit of a cliché and is a term that's been under a fair bit of scrutiny and criticism lately for other reasons.  I personally don't mind it, but others might. Consider perhaps using `excited` instead?